lib/hawkular/alerts/alerts_api.rb

```diff
@@ -210,6 +210,22 @@ def list_events(*criteria)
       query = generate_query_params(*criteria)
       http_get('/events' + query).map { |e| Event.new(e) }
     end
+
+    # Inject an event into Hawkular-alerts
+    # @param [String] id Id of the event must be unique
+    # @param [String] category Event category for further distinction
+    # @param [String] text Some text to the user
+    # @param [Hash<String,Object>] context Additional context information.
+    def create_event(id, category, text, context = {})
```

Comment:
I'm pretty sure there's some syntactic sugar to create the keys more nicely in the hash from method parameters, but I can't remember it now :(